pyproject.toml

```diff
@@ -89,3 +85,57 @@ ignore_missing_imports = true
 [[tool.mypy.overrides]]
 module = "prometheus_client.*"
 ignore_missing_imports = true
+
+[tool.ruff]
+line-length = 88
+exclude = [
+    ".bzr",
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".nox",
+    ".pants.d",
+    ".ruff_cache",
+    ".svn",
+    ".tox",
+    ".venv",
+    "__pypackages__",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    ".venv",
+]
+
+[tool.ruff.lint]
+select = [
+    # pycodestyle"
+    "E4",
+    "E7",
+    "E9",
+    "E501",
+    # Pyflakes
+    "F",
+    # isort
+    "I001"
+]
+
+# Allow fix for all enabled rules (when `--fix`) is provided.
+fixable = ["ALL"]
+unfixable = []
+
+[tool.ruff.format]
+# Like Black, use double quotes for strings.
+quote-style = "double"
```

Comment:
are all this settings not the default for ruff?

Reply:
Yes! I will remove them